common/browser.go

```diff
@@ -86,13 +88,13 @@ type browserVersion struct {
 
 // NewBrowser creates a new browser, connects to it, then returns it.
 func NewBrowser(
```

Comment:
I believe that the `Browser` shouldn't need both the `backgroundCtx` and `k6Ctx`, it should only work with one, `k6Ctx`. It's responsibility shouldn't be to interact with the chromium subprocess/connection through `browserProc` and `backgroundCtx`. I need to open a refactor issue for this.

Reply:
Refactored background context out, so this isn't totally true.